light-rpc/src/rpc.rs

```diff
@@ -40,4 +78,16 @@ mod tests {
             CONNECTION_POOL_SIZE,
         );
     }
+    #[test]
+    fn test_forward_transaction() {
+        let _light_rpc = LightRpc::new(
```

Comment:
i think the underscore before the variable name should be removed.

Reply:
Ho yeah forgot about that

Reply:
Anything else?